public/tab/tabConfig/popUpSignin.html

```diff
@@ -0,0 +1,77 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+        <style type="text/css">
+            .row {
+                padding: 8px 0px;
+            }
+    
+            .row input {
+                border: none;
+                border-bottom: 1px solid #4E9DC7;
+                background-color: transparent;
+                margin: 0 5px;
+                padding: 3px 5px;
+            }
+    
+            .row input:hover {
+                background-color: rgba(0,0,0,0.05);
+            }
+
+            .row input:active {
+                background-color: rgba(0,0,0,0.1);
+            }
+    
+            .row .label {
+                margin-bottom: 5px;
+            }
+    
+            .primary-button {
+                background: #4C509D;
+                border: 0.125rem solid;
+                border-color: transparent;
+                border-radius: 0.1875rem;
+                color: #FFFFFF;
+                cursor: pointer;
+                font: inherit;
+                height: 2rem;
+                min-width: 6rem;
+                padding: 0.25rem;
+                white-space: nowrap;
+                font-family: 'Segoe UI', Tahoma, Helvetica, Sans-Serif;
+            }
+        </style>    
+</head>
+<!--Purpose of this login html page is to show fake authentication when user clicks on SignIn Action -->
+<!--This login page will do nothing, except when user will click on Login page it will post back to Main controller -->
+<body>
+    <div class="row input-container">
+        <div>
+            User Name : <input type="text" />
+        </div>
+    </div>
+    <div class="row input-container">
+        <div>
+            Password : <input type="password" />
+        </div>
+    </div>
+    <div class="row input-container">
+        <div>
+            Magic Number : <input id="magicnumber" disabled="disabled" />
+        </div>
+    </div>
+
+    <input type="submit" class="primary-button" value="Submit" onClick="submit()">
+    <input type="submit" class="primary-button" value="Cancel" onClick="window.close();">
```

Comment:
Shouldn't this call notifyFailure?

Reply:
it's just to close the window with no user action, I think for sample scenario window.close() should also work.

Reply:
but aren't we teaching bad practices by doing this?